distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres.enso

```diff
@@ -20,10 +26,11 @@ type Postgres
        - host: The hostname of the database server (defaults to localhost).
        - port: The port of the database server (defaults to 5432).
        - database: The database to connect to. If empty, the default database will be used.
+       - schema: The schema to connect to. If empty, the default schema will be used.
```

Comment:
We connect to a database, but do we connect to a schema? Or do we choose a schema?

Reply:
Yes a JDBC connection has a selected schema (it may be not implemented if not supported - e.g. SQLite).